test/functionalTest/cases/2101_Subscription_is_not_created_with_attribute_names_equals/Subscription_is_not_created_with_attribute_names_equals.test

```diff
@@ -0,0 +1,409 @@
+# Copyright 2021 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Subscription_is_not_created_with_attribute_names_equals
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. POST /v2/subscriptions, to create subscription with N attributes with the same name in attrs and see error
+# 02. POST /v2/subscriptions, to create subscription with N attributes with the same name in condition and see error
+# 03. POST /v2/subscriptions, to create subscription with N attributes with the same name in exceptAttrs and see error
+# 04. POST /v2/subscriptions, to create subscription with N attributes with different name in attrs and see no error
+# 05. POST /v2/subscriptions, to create subscription with N attributes with different name in condition and see no error
+# 06. POST /v2/subscriptions, to create subscription with N attributes with different name in exceptAttrs and see no error
+# 07. POST /v2/subscriptions, to create subscription with N attributes with the same name in metadata and see error
+# 08. POST /v2/subscriptions, to create subscription with N attributes with different name in metadata and see no error
```

Comment:
I'd suggest the following ordering in steps (all errors first, all ok second):

```suggestion
# 01. POST /v2/subscriptions, to create subscription with N attributes with the same name in attrs and see error
# 02. POST /v2/subscriptions, to create subscription with N attributes with the same name in condition and see error
# 03. POST /v2/subscriptions, to create subscription with N attributes with the same name in exceptAttrs and see error
# 04. POST /v2/subscriptions, to create subscription with N attributes with the same name in metadata and see error
# 05. POST /v2/subscriptions, to create subscription with N attributes with different name in attrs and see no error
# 06. POST /v2/subscriptions, to create subscription with N attributes with different name in condition and see no error
# 07. POST /v2/subscriptions, to create subscription with N attributes with different name in exceptAttrs and see no error
# 08. POST /v2/subscriptions, to create subscription with N attributes with different name in metadata and see no error
```

(Of course, not only in this commented preamble but also the actual ordering of steps)

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4025/commits/81e67f5fb86e4e61d4879e6ee8924b88fdd5d5cb